google/cloud/storage/bucket.py

```diff
@@ -1021,6 +1023,35 @@ def list_notifications(self, client=None, timeout=_DEFAULT_TIMEOUT):
         iterator.bucket = self
         return iterator
 
+    def get_notification(self, notification_id, client=None, timeout=_DEFAULT_TIMEOUT):
+        """Get Pub / Sub notification for this bucket.
+
+        See:
+        https://cloud.google.com/storage/docs/json_api/v1/notifications/get
+
+        If :attr:`user_project` is set, bills the API request to that project.
+
+        :type notification_id: str
+        :param notification_id: The notification id to retrieve the notification configuration.
+
+        :type client: :class:`~google.cloud.storage.client.Client` or
+                      ``NoneType``
+        :param client: (optional) The client to use.  If not passed, falls back
+                       to the ``client`` stored on the current bucket.
+        :type timeout: float or tuple
+        :param timeout: (optional) The amount of time, in seconds, to wait
+            for the server response.
+
+            Can also be passed as a tuple (connect_timeout, read_timeout).
+            See :meth:`requests.Session.request` documentation for details.
```

Comment:
Can you add an example of how to use this helper method?

Reply:
Yes.